pkg/template/funcs.go

```diff
@@ -0,0 +1,54 @@
+package template
```

Comment:
we have all of these functions in the utils library...

Reply:
ok, removed it.